float64 time
string my_side
uint16 my_point
uint16 enemy_point
string[] target_names
string[] target_owner
uint16[] target_point